airbyte-integrations/connectors/source-mixpanel/source_mixpanel/manifest.yaml

```diff
@@ -44,6 +44,9 @@ definitions:
       - error_message_contains: "Query rate limit exceeded"
         action: RETRY
         error_message: Query rate limit exceeded.
+      - error_message_contains: "unknown error"
```

Comment:
To retry when there is a 500 error with this message which is retriable

Reply:
Is there a specific 500-level error code(s) we expect to throw this error? If so I think we should explicitly list any, similar to lines 32 and 38 of this file. My concern is using "unknown error" alone as a condition might be too broad of a net.

Reply:
I don't know. I known that sometimes the sync crash because of this error and I think it would better to retry it instead of stopping the sync.

I don't remember the exact message but it is something like "an unkown error has occured"

Reply:
The http code seems to be 500 in this case. We could add it too

Reply:
Gotcha, thanks for checking!